src-test/org/etools/j1939_84/controllers/part01/Part01Step10ControllerTest.java

```diff
@@ -50,6 +51,7 @@
 @RunWith(MockitoJUnitRunner.class)
 public class Part01Step10ControllerTest extends AbstractControllerTest {
 
+    public static final TimeUnit SECONDS = TimeUnit.SECONDS;
```

Comment:
Don’t do this.  You can import TimeUnit.SECONDS and it will appears in the code as SECONDS

Reply:
Updated [here.](https://github.com/SolidDesignNet/j1939-84/pull/20/commits/d16476b3c850a1fd258cc45895b9662e37fc1302)